java/pending_code_changes.txt

```diff
@@ -1 +1,3 @@
-
+Code changes:
+ - Made normalizeDiallableCharsOnly() API in PhoneNumberUtil public. This
```

Comment:
I really feel we should move this file up a level since it looks like it only applies to Java changes and that's how I use it, but it would be more useful if we also mentioned JS etc....